crates/ibc/src/core/ics04_channel/handler/send_packet.rs

```diff
@@ -1,3 +1,6 @@
+use crate::core::ics02_client::client_state::ClientStateValidation;
+use crate::core::ics02_client::client_state::Status;
+use crate::core::ics02_client::error::ClientError;
```

Comment:
As a small nit if you mind: nice to reorder `core` imports like this to be placed next to the other `core`s (all over the PR)

Reply:
My issue with this manual reordering is that however it got this way is because the Rust tools we use put it there; therefore, it will happen again. IMO, instead of doing some manual code formatting, we should either see if this feature is available in nightly `cargo fmt` (it might be, I think they have a bunch of nightly-only features), or wait for it be supported. Otherwise, we put the burden on us to manually check import statements (and fix them) for every PR, which is not a good use of our time.

Reply:
From what I see, the imports get automatically sorted alphabetically! As a later chore, perhaps we can order all import calls in that manner once. Afterward, whatever is added would be ok.